### PR TITLE
Add function to cut a triangle from an image

### PR DESCRIPTION
**Pull request by cjohansen**
The function cuts out a triangle from one of the four corners, and leaves the
other half of the image transparent.

### CODE REVIEW

File: src/fivetonine/collage/core.clj [hunk 1]
```diff
@@ -209,6 +210,35 @@
...
+
+  ```clj
+  (triangle image {:position :upper-left})
+  (triangle image {:position :lower-left})
+  (triangle image {:position :lower-right})
+  (triangle image {:position :upper-right})
+  ```"
+  [^BufferedImage image & [{:keys [position]}]]
+  (let [width (.getWidth image)
+        circle (BufferedImage. width width BufferedImage/TYPE_INT_ARGB)]
```
Comment (karls) [inline src/fivetonine/collage/core.clj]: Hey! Many thanks for all the PRs! 🤩

A minor thing, `circle` should probably really be `triangle`.

**Comment by cjohansen** [comment]
Haha, wow. Good job, me :) Well spotted! Fixed.

**Comment by karls** [comment]
Thanks!

Because I merged one of the PRs, they all conflict now with master. :-/ Do you mind rebasing?

**Comment by cjohansen** [comment]
Yeah, I noticed when I tried to merge them together on my own master earlier... I'll fix.

**Pull request merged**